ovn_k8s/modes/overlay.py

```diff
@@ -217,6 +237,16 @@ def delete_logical_port(self, event):
 
         vlog.info("deleted logical port %s" % logical_port)
 
+        if delete_network_policies:
```

Comment:
I did the change. The logic looks pretty much the same, but perhaps the code now it's a bit easier to debug.